Package.swift

```diff
@@ -154,6 +154,7 @@ let package = Package(
             name: "DatadogWebViewTracking",
             dependencies: [
                 .target(name: "DatadogInternal"),
+                .target(name: "TestUtilities")
```

Comment:
`DatadogWebViewTracking` should not depend on `TestUtilities`, I guess the change in L189 is enough 👍

Reply:
that was a mistake - already removed.